webapp_v2/src/App.tsx

```diff
@@ -22,7 +32,38 @@ function App() {
             <LoginWrapper>
               <MenuWrapper>
                 <Routes>
-                  <Route path="/studies" element={<Studies />} />
+                  <Route path="/studies" element={<Outlet />}>
+                    <Route index element={<Studies />} />
+                    <Route path=":studyId" element={<Outlet />}>
+                      <Route index element={<SingleStudy />} />
+                      <Route path="explore" element={<SingleStudy isExplorer />}>
+                        <Route path="modelization" element={<Modelization />}>
+                          <Route path="map" element={<Map />} />
+                          <Route path="area" element={<Area />} />
+                          <Route path="links" element={<Links />} />
+                          <Route path="bindingcontraint" element={<BindingContraint />} />
+                          <Route
+                            index
+                            element={<Navigate to="map" />}
+                          />
```

Comment:
set the element to <Map /> and do not navigate to it (it breaks the previous page system)